independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapMavenContext.java

```diff
@@ -460,9 +479,15 @@ private DefaultRepositorySystemSession newRepositorySystemSession() throws Boots
             }
             session.setMirrorSelector(ms);
         }
-        final String localRepoPath = getLocalRepo();
-        session.setLocalRepositoryManager(
-                getRepositorySystem().newLocalRepositoryManager(session, new LocalRepository(localRepoPath)));
+
+        final LocalRepositoryManager head = getRepositorySystem().newLocalRepositoryManager(session,
+                new LocalRepository(getLocalRepo()));
+        final List<LocalRepositoryManager> tail = new ArrayList<>();
+        for (final String tailPath : getLocalRepoTail()) {
+            tail.add(getRepositorySystem().newLocalRepositoryManager(session, new LocalRepository(tailPath)));
+        }
+
+        session.setLocalRepositoryManager(new ChainedLocalRepositoryManager(head, tail, getLocalRepoTailIgnoreAvailability()));
```

Comment:
Let's do this only if the tail was actually configured. Also for now let's not introduce public methods that are used only in this same class.
Thanks!

Reply:
Thanks! Done in ed070ce.